src/VSCodeExtension/src/generator-qsharp/app/index.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) Microsoft Corporation.
```

Updated guidance no longer suggests "All rights reserved" for new code.